code/extensions/ExtensibleSearchExtension.php

```diff
@@ -14,6 +14,18 @@ class ExtensibleSearchExtension extends Extension {
 		'results'
 	);
 
+	public function onAfterInit() {
```

Comment:
Adding this here we can include the search overlay on any page that has a searchbox and overlay div